packaging/windows/OpenRA.nsi

```diff
@@ -18,6 +18,7 @@
 !include "MUI2.nsh"
 !include "FileFunc.nsh"
 !include "WordFunc.nsh"
+!include "nsProcess.nsh"

```

Comment:
You need to install this NSIS plugin during the CI build. https://github.com/OpenRA/OpenRA/blob/bleed/appveyor.yml#L22